homework/english/raby_alexandr/lection1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
+2. I create my first homework in Github.
+3. Cannot connect to the specified server.
+4. We are debuging their code now. 
+5. She is developing new application at the moment.
+6. He disables his PC before going to bed.
+7. I am disconnecting its the network now.
+8. I am displaying the picture at the moment.
+9. She usually enables  this function.
+10. His chat bot executes different commands.
+11. She constantly emplements new functions in this chat bot.
+12. Are you installing this PC-game today?
+13. Do you help me to restore my program?
```

Comment:
It is better to use phrase "can u help me". And it is more correct

Reply:
Thank you for your comment